public/netsuke/assets/css/himotoshi.css

```
@@ -509,6 +541,142 @@ body {
+/* Long-form prose rendered from shared Markdown sources, where the generated
+   markup carries no utility classes of its own. */
+.hm-prose {
+  line-height: 2;
+}
+
+.hm-prose > * + * {
+  margin-top: 1.5rem;
+}
+
+.hm-prose h2 {
+  font-family: "Fraunces", serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)